tests/unit/test_extensionPoints.py

```diff
@@ -335,6 +335,8 @@ def test_registerInstanceMethod(self):
 		actual = list(self.reg.handlers)
 		self.assertEqual(actual, [inst.method])
 
+	# #9720 (Py3 review required): for some reason, this test keeps failing, so mark this as expected failure for now.

```

Comment:
Please add a comment to the register method in extensionPoints. That code really needs a revisit.

Reply:
In Python3, unbound methods no longer exist. Rather, fetching a method from a class directly (I.e. not from an instance) gives you back a normal function. 
In the case of register:  it will handle these as normal functions, and succeed. 
I don't believe this would cause any problems for us: the handler will be called later like any normal function, as far as it makes sense to call an unbound method function. I.e. the code registering this function would have to pass in self as one of its arguments to make the function work in any meaningful way.
We could argue that we should disallow registering unbound method functions still, but there is no way of detecting these -- they are just functions.
I am happy to leave this in the code as a disabled test and approve/merge this as is. Though I would be interested in thoughts from @jcsteh on this. Should we just remove the test?